util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisService.java

```diff
@@ -36,12 +38,17 @@ private SystematicSensitivityAnalysisService() {
 
     public static SystematicSensitivityAnalysisResult runAnalysis(Network network,
                                                                   Crac crac,
-                                                                  ComputationManager computationManager) {
+                                                                  ComputationManager computationManager,
+                                                                  RaoParameters parameters) {
         String initialVariantId = network.getVariantManager().getWorkingVariantId();
 
         Map<State, SensitivityComputationResults> stateSensiMap = new HashMap<>();
         Map<Cnec, Double> cnecFlowMap = new HashMap<>();
 
+        SearchTreeRaoParameters searchTreeRaoParameters = parameters.getExtensionByName("SearchTreeRaoParameters");
+        boolean dcMode = searchTreeRaoParameters.isDcMode();
+        boolean acToDcFallback = searchTreeRaoParameters.isAcToDcFallback();
```

Comment:
We are not sure that the searchTreeRaoParameters exist.
For instance, if we launch an "independent linear-rao", not within a search-tree-rao.

So concerning my first comment, I now think these parameters should be in RaoParameters (not SearchTreeRaoParameters).

Reply:
Do you mean in LinearRaoParameters ?

Reply:
I don't like putting it in an extension of a RaoParameters because you never know if the extension is here or not. I prefer the solution of the RaoParameters, or a dedicated SystematicSensitivityAnalysisParameters.